### DISCUSSION TITLE
Abschaltverhalten bei Stop-Batteriespannung

### DISCUSSION THREAD

**Discussion by Tommoca** [hoylabs/OpenDTU-OnBattery#1228] [General]
Hallo und erstmal vielen Dank an die Entwickler, wirklich tolles Projekt.
Ich hatte bisher eine DTU mit 4 MB und hab die gegen eine mit 16MB und 2024.08.18 getauscht.
Mit fällt ein Verhalten auf, was ich bei der alten Version nicht beobachten konnte.
Mein System:
24V LiFePo4-Akku, Shelly 3EM, Victron MPPT, Hoymiles HM-300
Einstellungen DPL:
Solar-Passthrough: aktiv
Verwenden Sie den Akku nachts, auch wenn er nur teilweise geladen ist: aktiv
Batterieschwellenwerte Start 27V / Stop 26V
Lastkorrekturfaktor: 0,0013

Die letzten Tage wurde der Akku nicht ganz voll geladen und hat Nachts den Stop-Schwellwert von 26V errreicht (entspricht ca 30%)
Der Hoymiles wird dann abgeschaltet (124 durch Fernsteuerung abgeschaltet)
Soweit OK
Dann wird er aber immer wieder, in mehr oder weniger kurzen Abständen, gestartet, läuft kurz an und wieder abgeschaltet.
Im Spannungsverlauf des Victron MPPT sieht man, dass der Akku sich in der Spannung etwas erholt, über die 26V geht und dann der Hoymiles wieder gestartet wird.
Passierte bisher bis zu 9 mal hintereinander.

Bei der alten DTU blieb der Hoymiles aus wenn die Spannung Nachts einmal unter die Stopschwelle gefallen ist. 
Die Werte von der alten DTU habe ich 1:1 zur neuen übernommen.
Macht das nicht mehr Sinn den Hoymiles auszulassen wenn er Nachts einmal die Stopschwelle erreicht hat?
Irgendwie habe ich bisschen die Befürchtung, dass das ständige An/Aus in kurzen Zeitabständen auf die Lebensdauer vom Hoymiles gehen könnte.

**Comment by spcqike** [comment]
> Irgendwie habe ich bisschen die Befürchtung, dass das ständige An/Aus in kurzen Zeitabständen auf die Lebensdauer vom Hoymiles gehen könnte.

ich finde es eher problematisch für den Akku, grade im unteren SoC Bereich.
Dem Hoymiles sollte es egal sein.

>  Verwenden Sie den Akku nachts, auch wenn er nur teilweise geladen ist: aktiv

bist du sicher, dass das vorher ebenfalls aktiv war?

**Reply by Tommoca** [reply]
Ja, war beim alten aktiv, ganz sicher. Der lief jetzt gut 1 Jahr. Ich habe auch mal diverse Tage im Frühjahr im MyStrom-Cockpit durchgesehen, da sehe ich solche Peaks absolut nicht. 
Akku habe ich weniger Befürchtungen, ich bin da ja noch im 30% Bereich. Da ich den in der Garage in einer isolierten Box mit 24V Heizfolie stehen hab, gehe ich da nicht weiter runter, falls die Heizung mal laufen muss. Die Folie ist natürlich über einen Victron BatteryProtect geschaltet der bei 20% abschaltet.
